### PR TITLE
Test:  improve test performance testing script

### PR DESCRIPTION
**Pull request by dtivel**
* support moving the test root folder (e.g.:  to another drive)
* support gathering logs (i.e.:  output from nuget.exe) from PerformanceTestRunner.ps1

### CODE REVIEW

File: scripts/perftests/PerformanceTestRunner.ps1 [hunk 1]
```diff
@@ -2,24 +2,21 @@ Param(
...
+    [string] $testRootFolderPath,
+    [string] $logsFolderPath,
     [int] $iterationCount = 3,
-    [switch] $SkipCleanup
+    [switch] $skipCleanup
 )
 
 . "$PSScriptRoot\PerformanceTestUtilities.ps1"
 
-If ([string]::IsNullOrEmpty($testFolderPath))
```
Comment (nkolev92) [inline scripts/perftests/PerformanceTestRunner.ps1:12]: Why remove the default?

Comment (dtivel) [reply scripts/perftests/PerformanceTestRunner.ps1:12]: The [call to GetNuGetFoldersPath](https://github.com/NuGet/NuGet.Client/pull/2760/files#diff-b716ac805ce14d3f74e55d4adc10e536R14) already applies a [default](https://github.com/NuGet/NuGet.Client/blob/23b76fef368db1df85d4ff5418bb66c25c499c29/scripts/perftests/PerformanceTestUtilities.ps1#L180).

Comment (nkolev92) [reply scripts/perftests/PerformanceTestRunner.ps1:12]: Ok, now that'd mean the default is not in temp anymore, but user profile.

**Review by nkolev92 — APPROVED** [review]

File: scripts/perftests/PerformanceTestRunner.ps1 (hunk 1 above)
Comment (dtivel) [reply scripts/perftests/PerformanceTestRunner.ps1:12]: Temp was the default only for sources while user profile was the default for packages caches/folders.  This change converges on one default root.  I don't have a strong preference for one over the other.  If you want me to change the default to temp, let me know.

Comment (nkolev92) [reply scripts/perftests/PerformanceTestRunner.ps1:12]: I think I kept them separate for 1 reason. 
It allowed me to run consecutive perf tests without re-downloading the sources. 

At this point I'm not 100% if this is still a concern in the scripts. 

I don't have a hard opinion either way, just think the scenario I am mentioning is important enough to consider.

Comment (dtivel) [reply scripts/perftests/PerformanceTestRunner.ps1:12]: The new `$skipRepoCleanup` option gives you that ability.

**Pull request merged**